### DISCUSSION TITLE
I would like to ask if there are any plans to add mind map/tree map function, thank you!

### DISCUSSION THREAD

**Discussion by XDao7** [patrykandpatrick/vico#486] [Ideas]
It looks like this：
![121](https://github.com/patrykandpatrick/vico/assets/50022005/e3cb12b2-abee-4342-944c-527b7fa2c916)

**Comment by patrickmichalik** [comment]
Hello! I’m afraid there are no such plans at the moment. We’ll be adding more chart types, but Vico is generally intended for numerical charts.
